### ISSUE TITLE
Wireguard service doesn't compile on windows

### ISSUE THREAD

**Issue by tadaskay** [mysteriumnetwork/node#1818]
Latest refactoring and nat hole punching hasn't been applied to `_windows` versions - they simply do not compile now. I've tried compiling natively on windows MinGW x64 7.2.0

**Comment by cvl** [comment]
Agreed, wireguard is only nice to have, shouldn't be a showstopper.
